popgen/stationarity.go

```diff
@@ -20,15 +20,24 @@ Equations from this thesis that are replicated here are marked.
 
 const IntegralBound float64 = 1e-12
 
+type LikelihoodFunction byte
+
+const (
+	Uncorrected LikelihoodFunction = 0
```

Comment:
This is totally fine (and even better in some ways) but just to share, I recently learned about the `iota` keyword for defining consts. If you wanted to, you could write this as 
```
const (
     Uncorrected LikelihoodFunction = iota
     Ancestral
     Derived
)
```

This is useful when you want discrete values for a variable, but don't care about the particular number it is assigned to. This is not any more efficient as far as I know (except typing less), but something to know exists.

Reply:
This is cool! Implemented in the next commit